FileTransactionStoreManager is never close so the seataFile could not be delete

- [x] I have searched the [issues](https://github.com/seata/seata/issues) of this repository and believe that this is not a duplicate.

### Ⅰ. Issue Description

FileTransactionStoreManagerTest test failure, the reason is that `FileTransactionStoreManager.currRaf` is holding the file and the `currRaf` never close. so when call `seataFile.delete` will return a value `false`.


### Ⅲ. Describe what you expected to happen

FileTransactionStoreManager provider a method to close currRaf, so that the file can be delete success.


### Ⅵ. Environment:

- JDK version : oracle jdk8
- OS : windows 10
- seata: 1.0.0-SNAPSHOT

the error is:
[INFO] Results:
[INFO]
[ERROR] Failures:
[ERROR]   FileTransactionStoreManagerTest.testBigDataWrite:68 expected: <true> but was: <false>
[ERROR]   FileTransactionStoreManagerTest.testFindTimeoutAndSave:114 expected: <true> but was: <false>
[INFO]
[ERROR] Tests run: 94, Failures: 2, Errors: 0, Skipped: 1

--- a/server/src/test/java/io/seata/server/store/file/FileTransactionStoreManagerTest.java
+++ b/server/src/test/java/io/seata/server/store/file/FileTransactionStoreManagerTest.java
@@ -78,6 +78,7 @@ public class FileTransactionStoreManagerTest {
         File seataFile = Files.newTemporaryFile();
         Method findTimeoutAndSaveMethod = FileTransactionStoreManager.class.getDeclaredMethod("findTimeoutAndSave");
         findTimeoutAndSaveMethod.setAccessible(true);
+        FileBasedSessionManager sessionManager = null;
         FileTransactionStoreManager fileTransactionStoreManager = null;
         try {
             List<GlobalSession> timeoutSessions = new ArrayList<>();
@@ -100,10 +101,11 @@ public class FileTransactionStoreManagerTest {
             SessionManager sessionManagerMock = Mockito.mock(SessionManager.class);
             Mockito.when(sessionManagerMock.findGlobalSessions(Mockito.any()))
                     .thenReturn(timeoutSessions);
-            fileTransactionStoreManager = new FileTransactionStoreManager(seataFile.getAbsolutePath(), sessionManagerMock);
+            fileTransactionStoreManager = new FileTransactionStoreManager(
+                seataFile.getAbsolutePath(), sessionManagerMock);
             Assertions.assertTrue((boolean) findTimeoutAndSaveMethod.invoke(fileTransactionStoreManager));
 
-            FileBasedSessionManager sessionManager = new FileBasedSessionManager(seataFile.getName(), seataFile.getParent());
+            sessionManager = new FileBasedSessionManager(seataFile.getName(), seataFile.getParent());
             sessionManager.reload();
             Collection<GlobalSession> globalSessions = sessionManager.allSessions();
             Assertions.assertNotNull(globalSessions);
@@ -118,6 +120,9 @@ public class FileTransactionStoreManagerTest {
             findTimeoutAndSaveMethod.setAccessible(false);
             if (null != fileTransactionStoreManager) {
                 fileTransactionStoreManager.shutdown();
+            }
+            if (null != sessionManager) {
+                sessionManager.destroy();
             }
             Assertions.assertTrue(seataFile.delete());
         }

--- a/server/src/test/java/io/seata/server/store/file/FileTransactionStoreManagerTest.java
+++ b/server/src/test/java/io/seata/server/store/file/FileTransactionStoreManagerTest.java
@@ -42,8 +42,9 @@ public class FileTransactionStoreManagerTest {
     @Test
     public void testBigDataWrite() throws Exception {
         File seataFile = Files.newTemporaryFile();
+        FileTransactionStoreManager fileTransactionStoreManager = null;
         try {
-            FileTransactionStoreManager fileTransactionStoreManager = new FileTransactionStoreManager(seataFile.getAbsolutePath(), null);
+            fileTransactionStoreManager = new FileTransactionStoreManager(seataFile.getAbsolutePath(), null);
             BranchSession branchSessionA = Mockito.mock(BranchSession.class);
             GlobalSession global = new GlobalSession();
             Mockito.when(branchSessionA.encode())
@@ -65,6 +66,9 @@ public class FileTransactionStoreManagerTest {
             BranchSession loadedBranchSessionB = (BranchSession) list.get(1).getSessionRequest();
             Assertions.assertEquals(branchSessionB.getApplicationData(), loadedBranchSessionB.getApplicationData());
         } finally {
+            if (null != fileTransactionStoreManager) {
+                fileTransactionStoreManager.shutdown();
+            }
             Assertions.assertTrue(seataFile.delete());
         }
     }
@@ -74,6 +78,7 @@ public class FileTransactionStoreManagerTest {
         File seataFile = Files.newTemporaryFile();
         Method findTimeoutAndSaveMethod = FileTransactionStoreManager.class.getDeclaredMethod("findTimeoutAndSave");
         findTimeoutAndSaveMethod.setAccessible(true);
+        FileTransactionStoreManager fileTransactionStoreManager = null;
         try {
             List<GlobalSession> timeoutSessions = new ArrayList<>();
             for (int i = 0; i < 100; i++) {
@@ -95,7 +100,7 @@ public class FileTransactionStoreManagerTest {
             SessionManager sessionManagerMock = Mockito.mock(SessionManager.class);
             Mockito.when(sessionManagerMock.findGlobalSessions(Mockito.any()))
                     .thenReturn(timeoutSessions);
-            FileTransactionStoreManager fileTransactionStoreManager = new FileTransactionStoreManager(seataFile.getAbsolutePath(), sessionManagerMock);
+            fileTransactionStoreManager = new FileTransactionStoreManager(seataFile.getAbsolutePath(), sessionManagerMock);
             Assertions.assertTrue((boolean) findTimeoutAndSaveMethod.invoke(fileTransactionStoreManager));
 
             FileBasedSessionManager sessionManager = new FileBasedSessionManager(seataFile.getName(), seataFile.getParent());
@@ -111,6 +116,9 @@ public class FileTransactionStoreManagerTest {
             });
         } finally {
             findTimeoutAndSaveMethod.setAccessible(false);
+            if (null != fileTransactionStoreManager) {
+                fileTransactionStoreManager.shutdown();
+            }
             Assertions.assertTrue(seataFile.delete());
         }
     }

